### ISSUE TITLE
Can not perform system update

### ISSUE THREAD

**Issue by TimeScroll** [ublue-os/bazzite#1323]
### Describe the bug

I get this bug.
![Screenshot_20240703_115648](https://github.com/ublue-os/bazzite/assets/26848467/d5a68b05-823c-4855-80e2-7fa54f167123)

### What did you expect to happen?

I expect the system update to work.

### Output of `rpm-ostree status`

```shell
State: idle
Deployments:
● ostree-image-signed:docker://ghcr.io/ublue-os/bazzite:stable
                   Digest: sha256:03f4b166f57323b6228982bb8b78297d88f6483faf5dfe45982498cd15722c80
                  Version: 40.20240625.0 (2024-06-27T01:59:48Z)

  ostree-image-signed:docker://ghcr.io/ublue-os/bazzite:stable
                   Digest: sha256:03f4b166f57323b6228982bb8b78297d88f6483faf5dfe45982498cd15722c80
                  Version: 40.20240625.0 (2024-06-27T01:59:48Z)
            LocalPackages: lact-0.5.4-0.x86_64
```

### Hardware

HP EliteDesk G5 mini

**Comment by schabpiotr** [comment]
The same happens for me on Steam Deck LCD

![image](https://github.com/ublue-os/bazzite/assets/33779042/35b42c4b-3016-483d-aa33-b465f01905b7)


rpm-ostree status output:
`State: idle
Deployments:
● ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck:latest
                   Digest: sha256:03a4738f0a7cb5fac998929d9f0349a16372d4b3ab836c39e2b652a7ddbaa2f9
                  Version: 40.20240528.0 (2024-05-28T17:05:50Z)
          LayeredPackages: chntpw coolercontrol liquidctl
                Initramfs: '"-I /etc/crypttab /etc/modprobe.d/amdgpu.conf /etc/modprobe.d/deck-blacklist.conf"' 

  ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck:latest
                   Digest: sha256:438ce4b4262669d3e36309b927d4e86e68c2f2440377a108b660863fd518a2f2
                  Version: 40.20240523.0 (2024-05-25T05:18:34Z)
          LayeredPackages: chntpw coolercontrol liquidctl
                Initramfs: '"-I /etc/crypttab /etc/modprobe.d/amdgpu.conf /etc/modprobe.d/deck-blacklist.conf"' 
`

**Comment by p5** [comment]
Hey both

We made some mistakes yesterday which caused updates to break.
Please can you follow the instructions in this Discourse post to remediate the issues.

TL;DR
We lost access to our signing keys so had to rotate them.
Users need to run the following command or manually perform the actions in [this script](https://github.com/ublue-os/config/blob/5c4144d38a2a61e44ca704ffdf2d50bbcb9ff7ed/scripts/fix-key-and-update.sh)

```
$ curl -sL https://fix.universal-blue.org/ | sudo bash 
```

Apologies for the inconvenience caused by this!

**Comment by schabpiotr** [comment]
> Hey both
> 
> We made some mistakes yesterday which caused updates to break. Please can you follow the instructions in [this Discourse post](https://universal-blue.discourse.group/t/important-announcement-regarding-system-updates-action-needed/2689) to remediate the issues.
> 
> TL;DR We lost access to our signing keys so had to rotate them. Users need to run the following command or manually perform the actions in [this script](https://github.com/ublue-os/config/blob/5c4144d38a2a61e44ca704ffdf2d50bbcb9ff7ed/scripts/fix-key-and-update.sh)
> 
> ```
> $ curl -sL https://fix.universal-blue.org/ | sudo bash 
> ```
> 
> Apologies for the inconvenience caused by this!

Trying this right now.
